pkg/apis/abac/v0/conversion.go

```diff
@@ -25,7 +25,9 @@ import (
 // but we don't want a client library (which must include types), depending on a server library
 const allAuthenticated = "system:authenticated"
 
-func Convert_v0_Policy_To_abac_Policy(in *Policy, out *abac.Policy, s conversion.Scope) error {
+// ConvertV0PolicyToAbacPolicy converts a V0-format policy into the current format,
+// and stores the result in the "out" buffer
+func ConvertV0PolicyToAbacPolicy(in *Policy, out *abac.Policy, s conversion.Scope) error {
```

Comment:
Thanks for the PR, but these method names must match the generated conversion method names, so this change will not work functionally

/close